Event_Planner/Classes/Flows/AppFlow.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  AppFlow.swift
+//  Event_Planner
+//
+//  Created by Edgars Baumanis on 25.02.19.
+//  Copyright © 2019. g. chili. All rights reserved.
+//
+
+import UIKit
+
+protocol FlowController {
+    var window: UIWindow?? {get}
```

Comment:
?? Optional optional should be fixed. Lets fix next time we have review session.